src/App/Platforms/Android/Autofill/AutofillHelpers.cs

```diff
@@ -347,7 +347,7 @@ public static RemoteViews BuildOverlayPresentation(string text, string subtext,
                 // InlinePresentation requires nonNull pending intent (even though we only utilize one for the
                 // "my vault" presentation) so we're including an empty one here
                 pendingIntent = PendingIntent.GetService(context, 0, new Intent(),
-                    AndroidHelpers.AddPendingIntentMutabilityFlag(PendingIntentFlags.OneShot | PendingIntentFlags.UpdateCurrent, true));
+                    AndroidHelpers.AddPendingIntentMutabilityFlag(PendingIntentFlags.OneShot | PendingIntentFlags.UpdateCurrent, false));
```

Comment:
🤔 Does this work as expected on Android 14-? Specially on minimum supported versions

Reply:
Alright sounds good 👍